### DISCUSSION TITLE
请问超参数组合实验的结果在哪里看到？

### DISCUSSION THREAD

**Discussion by KCAIED** [pykt-team/pykt-toolkit#205] [Q&A]
根据README中提供的下载链接（https://drive.google.com/drive/folders/1MWYXj73Ke3zC6bm3enu1gxQQKAHb37hz?usp=drive_link），其中只包含了每个模型在多个数据集上的参数组合情况，似乎未给出各组实验的最终结果，如AUC、ACC等

**Comment by zitaoliu** [comment]
你好，我们只提供我们搜索空间下的最优参数，不提供我们实验的每组参数的结果。您需要这个所有尝试过的参数组合的结果的目的是？

**Comment by KCAIED** [comment]
为我本人的研究提供对比实验数据（使用pylt库可以确保是在统一处理标准下的结果）；了解在各模型上，结果随参数组合调整而发生的变化趋势
----- 原始邮件 -----
发件人：Jerry Zitao Liu ***@***.***>
收件人：pykt-team/pykt-toolkit ***@***.***>
抄送人：yudingAIED ***@***.***>,  Author ***@***.***>
主题：Re:_[pykt-team/pykt-toolkit]_请问超参数组合实验的结果在哪里看到？_(Discussion_#205)
日期：2024年09月29日 19点40分


你好，我们只提供我们搜索空间下的最优参数，不提供我们实验的每组参数的结果。您需要这个所有尝试过的参数组合的结果的目的是？
